Packages/ca.tekly.datamodels/Editor/Models/ObjectEntry.cs

```diff
@@ -10,7 +10,8 @@ public class ObjectEntry
 		public GUIContent Id = new GUIContent() ;
 		public GUIContent Value = new GUIContent();
 		public string FullPath;
-		
+		public GUIContent FullPathGuid = new GUIContent();
```

Comment:
I don't understand the name FullPathGuid. Why is it a guid?

Reply:
Because my muscle memory automatically added the 'd'. Fixed.